### ISSUE TITLE
Will you support the CTC loss function?

### ISSUE THREAD

**Issue by yohacai** [tensorflow/tfjs#840]
Is the CTC loss function already supported by  tfjs-converte ?

**Comment by rthadur** [comment]
Automatically closing due to lack of recent activity. Please update the issue when new information becomes available, and we will reopen the issue. Thanks!

**Comment by kouohhashi** [comment]
Hi, I want to train speech recognition feature with tensorflow.js. and CTC loss function is key for that.
I hope there are some more people interested in CTC loss...

**Comment by fahimsun** [comment]
I need to train Speech Recognition and CTC work as loss function.

I hope to see this implemented in Tensorflow.js

**Comment by dsmilkov** [comment]
Hi everyone, this is a great issue for an external contribution. I'm happy to provide guidance.

**Comment by sbhamad** [comment]
@dsmilkov i would like to add support for ctc in tfjs, but i am clueless and do not know where to start. 

I wonder if c++ non parallel implementation should be compilable to wasm and be usable from tfjs, rather than writing a js implementation. Nonetheless, i dont mind writing an implementation in js but not sure how to go on about that. 

Thanks

**Comment by IvanVishnevskiy** [comment]
And I need ctc loss for OCR. I doubt I can help with that as I don't really understand the https://www.tensorflow.org/api_docs/python/tf/nn/ctc_loss's source code nor can I implement it by the original paper, unfortunately.

**Comment by HarshalRohit** [comment]
Are there any progress on this?
